src/app/domains/transactions/domains.transactions.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  declarations: [],
+  imports: [
+    CommonModule,
+  ],
+})
+export class DomainsTransactionsModule { }
```

Comment:
Module seems to be obsolete

Reply:
It's used. Just made mistake in imports in domains.module.ts

Reply:
Still does not seem to be used. Gateways are anyway registered against root module. You are not declaring any components here. So just empty module imported into another module.